DfciPkg/Library/DfciRecoveryLib/DfciRecoveryLib.c

```diff
@@ -129,6 +129,16 @@ GetRecoveryChallenge (
                                 &NewChallenge->Nonce.Bytes[0]
                                 );
         DEBUG ((DEBUG_VERBOSE, "%a: GetRNG(Hash256) = %r\n", __FUNCTION__, Status));
+        DEBUG ((DEBUG_INFO, "If non of above available, will use default RNG algorithm\n"));
+        if (EFI_ERROR (Status)) {
+          Status = RngProtocol->GetRNG (

```

Comment:
The concern with this approach is that `DfciPkg` is a generic package. How does it ensure that a "default" algorithm for any given RNG Protocol instance is actually "secure"?

Reply:
@makubacki  Could this be modified instead to add support for the gEfiRngAlgorithmArmRndr algorithm?
ARM guarantees BSI AIS-31 and NIST SP 800-90B compliance.
https://developer.arm.com/documentation/100685/0000/Overview-of-Arm-True-Random-Number-Generator--TRNG-